kubernetes/Dockerfile

```
@@ -0,0 +1,7 @@
+FROM mcr.microsoft.com/dotnet/framework/aspnet:4.8-windowsservercore-ltsc2019
+
+COPY windows_exporter-0.13.0-amd64.exe C:
```

In order not to depend on local files, you could download it from the Internet:

```suggestion
ADD https://github.com/prometheus-community/windows_exporter/releases/download/v0.13.0/windows_exporter-0.13.0-amd64.exe /windows_exporter-0.13.0-amd64.exe
```